formidable/forms/field_builder.py

```diff
@@ -89,11 +89,9 @@ def get_help_text(self):
         return self.field.help_text
 
     def get_validators(self):
```

Comment:
You can keep going with this idea.

Like @aRkadeFR said, you can wrap  ( I understand correctly) gen_validators with list(gen_validators()) on get_validators. Same thing for build_rules.